client/blocks/reader-unsubscribed-feeds-search-list/reader-unsubscribed-feed-item.tsx

```diff
@@ -42,6 +44,8 @@ const ReaderUnsubscribedFeedItem = ( {
 	title,
 }: ReaderUnsubscribedFeedItemProps ) => {
 	const translate = useTranslate();
+	const currentUser = useSelector( getCurrentUser );
+	const isEmailVarified = currentUser?.email_verified;
```

Comment:
Nit - small typo for the variables.

```suggestion
	const isEmailVerified = currentUser?.email_verified;
```

Reply:
Thanks for catching the typo @Aurorum!